docs/Operators.md

```diff
@@ -7226,6 +7314,93 @@ expect(node, inputs=[data], outputs=[transposed],
 </details>
 
 
+### <a name="Unique"></a><a name="unique">**Unique**</a>
+
+  Returns the unique scalar elements of the input tensor as a 1-D tensor. The
+  order of the elements in the output tensor is unspecified.
```

Comment:
Order need to be specified, how we will match the output between frameworks.

Reply:
Here TF implementation: https://www.tensorflow.org/api_docs/python/tf/unique

From the above: "This operation returns a tensor y containing all of the unique elements of x sorted in the same order that they occur in x.", Why don't we do the same?

Reply:
@gramalingam specifying this as unsorted with an auxiliary `Sort` operator is the more general case. Concretely:

- Front-end with sorted Unique operator: emit a Unique and a Sort operator to ONNX
- Front-end without a sorted Unique operator: emit a Unique
- Back-end with a sorted Unique operator: Fuse `Sort` into `Unique` with a simple peephole pass and emit the sorted operator. In the case where we find an unsorted `Unique` operator, we're still fine since the ordering semantics were unspecified and the network description must be valid regardless of the order. The backend is free to emit the sorted Unique op in place of the unsorted one
- Back-end without a sorted Unique operator: Can still accept Unique without a corresponding `Sort` operator. Fault on `Sort`

In the case where we specify the ordering in this schema

- Front-end with sorted Unique operator: emit Unique
- Front-end without a sorted Unique operator: emit a Unique *changing ordering semantics*
- Back-end with a sorted Unique operator: emit sorted Unique
- Back-end without a sorted Unique operator: *cannot accept Unique at all, unless you lower to separate Unique + Sort ops, as is specified here*

Both front-ends and back-ends may choose to implement `Unique` to be unsorted, for example for efficiency reasons. An example is caffe2.

Having a separate `Sort` is more general anyway and will likely find uses other than in tandem with `Unique`

Reply:
@jamesr66a that mean the Unique would match between frameworks? Would the test only verify that the elements are in or not?